lib/tasks/do_everything.rake

```diff
@@ -0,0 +1,11 @@
+namespace :do do
```

Comment:
Whoops, wondered where this has gone - popped this in to test new analytics releases, obvs. can be expanded on and in the end, pass around the aggregated meter collection...